.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Description
+
+_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue._
+
+## Merge requirements
+
+Please take into account the following non-code changes that you may need to make with your PR:
+
+- [ ] CHANGELOG entry for user-facing changes
+- [ ] Updated the relevant documentation
```

Comment:
I would also add a section to list issues fixed by the PR. Something like **Fixes**

Reply:
The following is in the `Description` section:

> If it fixes a bug or resolves a feature request, be sure to link to that issue